sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Compute Resources.")]
 
-[assembly: AssemblyVersion("35.0.0.0")]
-[assembly: AssemblyFileVersion("35.0.0.0")]
+[assembly: AssemblyVersion("35.1.0.0")]
```

Comment:
You don't have to update AssemblyVersion, because you are incrementing a minor version